Userland/Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -151,7 +151,9 @@ ALWAYS_INLINE OpCode* ByteCode::get_opcode_by_id(OpCodeId id) const
     if (id > OpCodeId::Last)
         return nullptr;
 
-    return const_cast<OpCode*>(s_opcodes[(u32)id]->set_bytecode(*const_cast<ByteCode*>(this)));
+    auto* opcode = &*s_opcodes[(u32)id];
+    opcode->set_bytecode(*const_cast<ByteCode*>(this));
+    return opcode;
```

Comment:
Not entirely sure what purpose this (removing the return value) serves, doesn't feel very perf related?

Reply:
Well, it's not perf-related but most of our other setters don't return `*this` so I figured I'd fix that while I was at it. Would you prefer this in a separate PR?

Reply:
Nah, it's alright as it is in this PR.
I was just curious if it somehow had a perf impact.

Reply:
Nah, it probably doesn't - or at least I didn't bother to measure the difference if there is any. :)